### ISSUE TITLE
API page for preview images is crashing

### ISSUE THREAD

**Issue by bhilbert4** [spacetelescope/jwql#1044]
https://jwql-test.stsci.edu/api/1068/preview_images/

- [x] Crashes with an error that it's trying to call split() on a tuple.

**Comment by bhilbert4** [comment]
Error is in get_preview_images_by_proposal().

- [x] I'm also seeing the same error when trying https://jwql-test.stsci.edu/api/1068/filenames/

In that case, the error is in get_filenames_by_proposal

**Comment by bhilbert4** [comment]
- [ ] The instrument/preview_images and 
- [ ] instrument/thumbnails API views also fail with:

key error: 'nircam'
data_containers.py, line 638, in mast_query_filenames_by_instrument

**Comment by bhilbert4** [comment]
- [x] /api/jw01538154001_21101_00003_nrca3/filenames/ fails with a similar error to that in the proposal/filename view above. Line 697 in data_containers

- [x] /api/jw01538154001_21101_00003_nrca3/preview_images also fails with the same error in line 978 of data_containers.py

**Comment by bhilbert4** [comment]
I think we should disable the list of all preview images and all thumbnails for an instrument. These lists will already contain hundreds of thousands of entries and will only be getting longer. Using the API, users can get a list of proposals per instrument, and then get a list of preview images or thumbnails per proposal, so they can build up a list of all preview images or thumnbnail files per instrument if they really want to.

When I tried retrieving a list of all preview images for nircam, I got back a bad gateway error. Same with the thumbnail list.
